cmake/FindSIP.cmake

```diff
@@ -42,6 +42,7 @@ ELSE(SIP_VERSION)
     STRING(REGEX REPLACE ".*\ndefault_sip_dir:([^\n]+).*$" "\\1" SIP_DEFAULT_SIP_DIR ${sip_config})
     STRING(REGEX REPLACE ".*\nsip_inc_dir:([^\n]+).*$" "\\1" SIP_INCLUDE_DIR ${sip_config})
     STRING(REGEX REPLACE ".*\nsip_mod_dir:([^\n]+).*$" "\\1" SIP_MOD_DIR ${sip_config})
+    STRING(REGEX REPLACE ".*\nsip_module_dir:([^\n]+).*$" "\\1" SIP_MODULE_DIR ${sip_config})
```

Comment:
you would have to also change https://github.com/qgis/QGIS/blob/master/cmake/FindSIP.py#L46 to reflect this.
Can you group #7365  and make a single PR?

Reply:
sure, let me try to clone the repo :)